### ISSUE TITLE
Typo in the pre-load mob stages script and a error with the furnace

### ISSUE THREAD

**Issue by notWaterGuy12** [hoklai1997/Coven-Undone#2]
Currently, the Mob Stages zs file in Pre-Load has a typo on scarecrow. Instead of mod_lavacow:scarecrow, it is mod_lavacow:scareow.

Also printed in chat once a world is loaded is: 
"WARNING: Furnace recipe from 1xtile.air@32767 to 1xtile.air@0 has an empty stack."

Recently found your pack, excited to be giving it a try. Looks like a lot of effort has gone into it.

**Comment by hoklai1997** [comment]
Thanks for finding that! Ill be pushing the change pretty soon for the scarecrow and am still figuring out the furnace issue, glad you’re interested in the pack, its still a bit early for it but im trying to get it to a releasable state as soon as possible!

**Comment by notWaterGuy12** [comment]
Awesome! One little thing I'm noticing is that aqua creepers make a lot of bodies of water very challenging to approach. If there is a way to purely disable the terrain damage of their explosions that would be an ideal change. If not, I'd suggest something like increase primalcore sea creature spawns or maybe more spawns of swarmers/pirahnas from undead rising. Overall, very cool pack so far, love to see packs do a more primal start than just go get stone and immediately get iron

**Comment by hoklai1997** [comment]
Yeah, aqua creepers have been challenging, let me know if they are as difficult in the most recent update, I tuned down theyre spawning quite a bit, and yeah, ill probably be tuning up piranhas or something in the future.

**Comment by notWaterGuy12** [comment]
When will this update be out? The latest file on Curseforge is from May. I found their spawning fairly frequent, enough for me to approach a coastline and find around 4 charging at me

**Comment by hoklai1997** [comment]
Yeah, ive had that experience a number of times, should already be up, from ~2 hrs ago

**Comment by notWaterGuy12** [comment]
strange, still shows the version from May as the most recent. must not be up on their end yet, I will definitely be giving it a try once it is up
quick edit: i was being silly and not checking for alpha version, will be giving it a try
